Storage/docu/docu_3_code.cc

```diff
@@ -519,29 +547,74 @@ TEST(StorageDocumentation, IfUnmodifiedSince) {
     EXPECT_EQ(412, static_cast<int>(response.code));
   }
 
-  const auto update_time = std::chrono::microseconds(1467374400000000);  // `Fri, 01 Jul 2016 12:00:00 GMT`.
+  auto update_time = std::chrono::microseconds(1467374400000000);  // `Fri, 01 Jul 2016 12:00:00 GMT`.
```

Comment:
Same comments as above -- why not `const`, and perhaps parse the string and `EXPECT_EQ` its numerical value?